vnext/Scripts/JustMyXaml.ps1

```diff
@@ -0,0 +1,20 @@
+# This script enables or disables VS 2019's JustMyXaml feature
+[CmdletBinding()]
+param([bool]$Enable)
+
+if (!([bool](([System.Security.Principal.WindowsIdentity]::GetCurrent()).groups -match "S-1-5-32-544"))) {
+    throw "Please run this script elevated";
```

Comment:
Is there a HKCU version of this you can do if not elevated.  Its annoying that so much of our stack keeps trying to push elevation.

Also if we are going to try to do add this to doctor in the future, maybe it shouldn't throw, but just note that it couldn't do it due to lack of elevation and gracefully exit?

Reply:
there isn't - you have the option of mounting a hive directly under HKLM or directly under HKU but you need permissions to create a key under HKU, so effectively you have to be elevated if going through reg.exe. Apps can use the win32 api to load the hive and you don't need to create a virtual key for that since you just deal with the `HANDLE`

Throwing is actually the right way to set the exit code when running this in its own process, otherwise you can't distinguish success from failure.